server/Cargo.toml

```diff
@@ -47,8 +47,9 @@ tokio-util = {git = 'https://github.com/mithril-security/tokio-sgx', tag = "toki
 tonic-rpc = {git = "https://github.com/mithril-security/tonic-rpc.git", branch = "main"}
 tower = {git = 'https://github.com/mithril-security/tower-sgx', branch = "tower-sgx"}
 tracing-core = {git = 'https://github.com/mithril-security/tracing-sgx', branch = "tracing-core-sgx"}
-tract-core = {git = "https://github.com/mithril-security/tract-sgx-xargo.git", tag = "0.16.6-pre-interpolation-nearest-bump-tar"}
-tract-onnx = {git = "https://github.com/mithril-security/tract-sgx-xargo.git", tag = "0.16.6-pre-interpolation-nearest-bump-tar"}
+tract-core = {git = "https://github.com/mithril-security/tract-sgx-xargo.git", tag = "0.17.1-pre"}
```

Comment:
did you intend to bump tract to 0.17 or is that just a mistake?
I would prefer to keep it at 0.16 for now and bump it to 0.17 on another PR afterwards :)